CMakeLists.txt

```diff
@@ -153,7 +153,7 @@ if(GIT_FOUND)
         math(EXPR VERSION_PATCH "${VERSION_PATCH} + 1")
     endif()
     add_definitions(-DV_MAJOR=${VERSION_MAJOR} -DV_MINOR=${VERSION_MINOR}
-                    -DV_PATCH=${VERSION_PATCH} -DV_LABEL=${VERSION_LABEL})
+                    -DV_PATCH=${VERSION_PATCH} -DV_LABEL="${VERSION_LABEL}")
```

Comment:
Please do not use `add_definitions` to introduce V_LABEL to every sourcefile.
V_LABEL is going to change every time the latest commit hash changes, and then every file will recompile because the macro changed.

Please do what the SHA1_HASH macro does instead, where it is added to a specific C file (or maybe multiple files, in this case?).

as V_LABEL is used in multiple sourcefiles, you might consider a new `.c` file that exports a `extern char const *const openomf_version_label;` in `.h`?